### PR TITLE
Create owasp-vulnerableapp.yml

### PR DESCRIPTION
**Pull request by preetkaran20**
I am the Project leader for Owasp-VulnerableApp and wanted to give my project more visibility and help to grow it.
Please review the PR.

thanks,
Karan

### CODE REVIEW

**Comment by preetkaran20** [comment]
Hi Team,

I have added the entries using up-for-grabs tool but that has not mentioned any validations like : is not allowed or space in tags is not allowed, lower-case letters are only allowed. can you guys please update the tool too to validate and also give some indication while taking the input so that raising PR's becomes more easier.

thanks,
Karan

**Comment by ritwik12** [comment]
@preetkaran20 By tool did you meant `Yeoman Generator`?

File: _data/projects/owasp-vulnerableapp.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - javascript
+  - payload-testing
+  - learn-security
+  - owasp-zap
+  - burp
+  - learn-hacking
+  - dast
+upforgrabs:
+  name: good first issue
+  link: https://github.com/SasanLabs/VulnerableApp/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```
Comment (ritwik12) [inline _data/projects/owasp-vulnerableapp.yml]: @preetkaran20 Please use this instead https://github.com/SasanLabs/VulnerableApp/labels/good%20first%20issue

File: _data/projects/owasp-vulnerableapp.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Owasp VulnerableApp
+desc: As Web Applications are becoming popular these days, there comes a dire need to secure them. Although there are several Vulnerability Scanning Tools, however while developing these tools, developers need to test them. Moreover, they also need to know how well is the Vulnerability Scanning tool performing. As of now, there are little or no such vulnerable applications existing for testing such tools. There are Deliberately Vulnerable Applications existing in the market but they are not written with such an intent and hence lag extensibility, e.g. adding new vulnerabilities is quite difficult. Hence, the developers resort to writing their own vulnerable applications, which usually causes productivity loss and the pain to rework.
```
Comment (ritwik12) [inline _data/projects/owasp-vulnerableapp.yml]: @preetkaran20 I recommend having a short and crisp description. Can we please work on this?

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by preetkaran20** [comment]
> @preetkaran20 By tool did you meant `Yeoman Generator`?

Yes

**Comment by preetkaran20** [comment]
Hi @ritwik12 ,

I have done the suggested change. Please review.

thanks,
Karan

**Comment by ritwik12** [comment]
> > @preetkaran20 By tool did you meant `Yeoman Generator`?
> 
> Yes

@preetkaran20 Please feel free to open an issue with what you faced and what should be there. We would be happy to get that sorted.

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@preetkaran20 Thanks!!

**Pull request merged**